app/models/person.rb

```diff
@@ -4,4 +4,13 @@ class Person < ApplicationRecord
 
   # Using UUIDs as the record IDs. We can't trust sequential ordering by ID
   default_scope { order(created_at: :asc) }
+
+  def full_name
+    [first_name, last_name].compact.join(' ')
+  end
+
+  # Temporarily until we've migrated all records to the new first/last name format
+  def full_name=(val)
```

Comment:
What's this doing? Is it setting the full name to the last name value when called?

Reply:
Yes, as we repurposed the existing `name` (never really used) and `full_name` DB fields to be `first_name` and `last_name` respectively, and all the current forms were using `full_name`, instead of renaming everywhere this is a way of 'aliasing' any assignments of `full_name` to go the `last_name` field instead.

We will need this code for some time until all forms are using the split format first/last name, then we will not need it anymore and can be removed (part of a bigger cleaning up really).